collect_app/src/main/java/org/odk/collect/android/mainmenu/MainMenuActivity.kt

```diff
@@ -329,12 +329,16 @@ class MainMenuActivity : LocalizedActivity() {
 
         val formSavedSnackbarDetails = mainMenuViewModel.getFormSavedSnackbarDetails(uri)
 
-        formSavedSnackbarDetails?.let { it ->
+        formSavedSnackbarDetails?.let {
             SnackbarUtils.showLongSnackbar(
                 binding.root,
                 getString(it.first),
-                action = SnackbarUtils.Action(getString(it.second)) {
-                    formLauncher.launch(FormFillingIntentFactory.editInstanceIntent(this, uri))
+                action = if (it.second == null) {
+                    null
+                } else {
+                    SnackbarUtils.Action(getString(it.second!!)) {
```

Comment:
You could avoid the `!!` with a `let` or `apply` here.

Reply:
Fixed.